rtgui/partialpastedlg.h

```diff
@@ -67,7 +67,7 @@ class PartialPasteDlg : public Gtk::Dialog
 
     // options in color:
     Gtk::CheckButton* icm;
-    Gtk::CheckButton* gam;
```

Comment:
Since version control takes care of reviving stuff, I suggest removing the variable instead of commenting it out if it is not used anymore.